multigeiger/speaker.cpp

```diff
@@ -138,8 +144,9 @@ void IRAM_ATTR tick(bool high) {
     sequence[6] = led_tick ? (high ? 0 : -1) : -1;
     sequence[7] = 0;  // END
   } else
-    sequence = NULL;
+    return;
```

Comment:
yeah, resetting isr_tick_sequence to NULL is done already at the end of playing the tick sound, so this is not required here.

i somewhat dislike returning from the else-block though.
if this change is kept, maybe rather move the block below into the if-block?

Reply:
Alright, moved the assignment block into the if block

Reply:
\o/